pages/docs/reference/whatsnew14.md

```diff
@@ -500,3 +586,9 @@ Complete the initial configuration, and when you add a new dependency to `cocoap
 The new dependency will be added automatically. No additional steps are required.
 
 Learn [how to add dependencies](native/cocoapods.html).
+
+### mimalloc memory allocator
+
+To improve the speed of object allocation, Kotlin/Native now offers the [mimalloc](https://github.com/microsoft/mimalloc)
+memory allocator as an alternative to the system allocator. mimalloc works up to two times faster on some benchmarks.
+Currently, the usage of mimalloc in Kotlin/Native is experimental; you can switch to it using the `-Xallocator=mimalloc` compiler option.
```

Comment:
May be also provide here the gradle description as in previous parts?
`binaries.all {
            freeCompilerArgs += "-Xallocator=mimalloc"
        }`

Reply:
Thanks, added an example